src/features/vote/PastViewDetailsModal.tsx

```diff
@@ -87,21 +93,16 @@ const _PastViewDetailsModal: ForwardRefRenderFunction<
       >
         <ModalWrapper>
           <MiniHeader>Proposal</MiniHeader>
-          <Proposal>{proposal}</Proposal>
-          <MiniHeader>Description</MiniHeader>
+          <Proposal>{isUmip ? umip?.title : proposal}</Proposal>
+
           <Description>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi eu
-            fringilla nunc. Sed mi arcu, consequat sed magna sed, auctor blandit
-            velit. Aliquam tincidunt, tortor vestibulum tempor tincidunt, lorem
-            velit dapibus purus, in lacinia velit lectus eget libero. Phasellus
-            sit amet lacinia ipsum, sed sollicitudin sapien. Integer et lacinia
-            nulla. Suspendisse ultrices, nisl vel egestas aliquam, nisi mi
-            imperdiet quam, ut faucibus diam ante ut elit. Vivamus venenatis a
-            purus nec vehicula. Cras mollis vel ligula nec vulputate. Integer
-            vehicula molestie sapien, eu dapibus metus auctor in. Nullam viverra
-            urna odio, sit amet lobortis metus interdum id. Nulla enim justo,
-            eleifend in metus in, eleifend dapibus ante. Donec nec egestas
-            lacus.
+            <ReactMarkdown
+              components={{
+                h1: MiniHeader,
+              }}
+            >
+              {umip?.description}
```

Comment:
Should this have some kind of place holder, like "No description" or will it look fine being empty

Reply:
Not use, but someone from the content team will

Reply:
Also, I've added a fallback in case there is no description found.